FlowCrypt/Functionality/Services/EKMVcHelper.swift

```diff
@@ -96,7 +100,8 @@ final class EKMVcHelper: EKMVcHelperType {
         in viewController: UIViewController,
         context: AppContextWithUser,
         keyDetail: KeyDetails,
-        passPhrase: String
+        passPhrase: String,
+        storageMethod: StorageMethod
```

Comment:
let's call it `passPhraseStorageMethod` else it's confusing - it may seem like it's about the key, but it's about the pass phrase